javascript/ql/src/Security/CWE-178/CaseSensitiveMiddlewarePath.ql

```diff
@@ -0,0 +1,121 @@
+/**
+ * @name Case-sensitive middleware path
+ * @description Middleware with case-sensitive paths do not protect endpoints with case-insensitive paths
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 7.3
+ * @precision high
+ * @id js/case-sensitive-middleware-path
+ * @tags security
+ *       external/cwe/cwe-178
+ */
+
+import javascript
+
+/**
+ * Converts `s` to upper case, or to lower-case if it was already upper case.
+ */
+bindingset[s]
+string invertCase(string s) {
```

Comment:
In isolation, I find the naming confusing here. If `s` is mixed case, it will be upper-cased, that does not sound like an inversion.

Reply:
Renamed to `toOtherCase`